pennylane/circuit_drawer/charsets.py

```diff
@@ -0,0 +1,199 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the different character set classes supported by PennyLane's CircuitDrawer.
+"""
+import abc
+
+
+class CharSet(abc.ABC):
+    """Charset base class."""
+
+    # pylint: disable=too-few-public-methods
+
+    @property
+    @abc.abstractmethod
+    def WIRE(self):
+        """The string used to represent a wire."""
+
+    @property
+    @abc.abstractmethod
+    def MEASUREMENT(self):
+        """The string used to represent a measurement at the end of a circuit."""
+
+    @property
+    @abc.abstractmethod
+    def TOP_MULTI_LINE_GATE_CONNECTOR(self):
+        """The string used to connect to the topmost wire of a multi-wire gate."""
+
+    @property
+    @abc.abstractmethod
+    def MIDDLE_MULTI_LINE_GATE_CONNECTOR(self):
+        """The string used to connect to the a wire of a multi-wire gate that is neither topmost nor bottommost."""
+
+    @property
+    @abc.abstractmethod
+    def BOTTOM_MULTI_LINE_GATE_CONNECTOR(self):
+        """The string used to connect to the bottommost wire of a multi-wire gate."""
+
+    @property
+    @abc.abstractmethod
+    def EMPTY_MULTI_LINE_GATE_CONNECTOR(self):
+        """The string used to symbolize unaffected wires when connecting multi-wire gates."""
+
+    @property
+    @abc.abstractmethod
+    def CONTROL(self):
+        """The string used to represent the controlling wires of an operation."""
+
+    @property
+    @abc.abstractmethod
+    def LANGLE(self):
+        """The string used to represent a left angle bracket."""
+
+    @property
+    @abc.abstractmethod
+    def RANGLE(self):
+        """The string used to represent a right angle bracket."""
+
+    @property
+    @abc.abstractmethod
+    def VERTICAL_LINE(self):
+        """The string used to represent a vertical line."""
+
+    @property
+    @abc.abstractmethod
+    def CROSSED_LINES(self):
+        """The string used to represent crossed lines."""
+
+    @property
+    @abc.abstractmethod
+    def PIPE(self):
+        """The string used to represent a vertical line of text height."""
+
+    @property
+    @abc.abstractmethod
+    def OTIMES(self):
+        """The string used to represent a tensor product operation."""
+
+    @staticmethod
+    @abc.abstractmethod
+    def to_superscript(num):
+        """Convert the given number to a superscripted string."""
+
+    @staticmethod
+    @abc.abstractmethod
+    def to_subscript(num):
+        """Convert the given number to a subscripted string."""
+
+
+class UnicodeCharSet(CharSet):
+    """Charset for CircuitDrawing made of Unicode characters."""
+
+    # pylint: disable=too-few-public-methods
+
+    WIRE = "─"
+    MEASUREMENT = "┤"
+    TOP_MULTI_LINE_GATE_CONNECTOR = "╭"
+    MIDDLE_MULTI_LINE_GATE_CONNECTOR = "├"
+    BOTTOM_MULTI_LINE_GATE_CONNECTOR = "╰"
+    EMPTY_MULTI_LINE_GATE_CONNECTOR = "│"
+    CONTROL = "C"
+    LANGLE = "⟨"
+    RANGLE = "⟩"
```

Comment:
For me, these render not so well, and it is hard to tell the difference between them and the pair of ``(`` , ``)``. Could ``<`` and ``>`` work here as well?

Reply:
For me they render fine. And I don't like those `<`, `>` because they don't really look like angled brackets because of their height